typings/custom-hash-node-map-defs.d.ts

```diff
@@ -0,0 +1,11 @@
+interface CustomHashNodeMap extends NamedNodeMap {
```

Comment:
Oh I think I've been referring this as `CustomHashNodeMap` whole time. Sorry, it should be `CustomNamedNodeMap`